web/templates/settings/admin_invites_list.hbs

```diff
@@ -33,12 +33,23 @@
     </td>
     <td class="actions">
         {{#unless is_multiuse}}
-        <button class="button rounded small resend btn-warning" {{#if disable_buttons}}disabled="disabled"{{/if}} data-invite-id="{{id}}">
-            {{t "Resend" }}
-        </button>
+            <button
+              class="button rounded small resend-invite btn-warning tippy-zulip-tooltip"
+              {{#if disable_buttons}}disabled="disabled"{{/if}}
+              data-invite-id="{{id}}"
+              data-tippy-content="{{t 'Resend'}}"
```

Comment:
I am not sure whether this reformatting help in readability. Since, I am not sure, I would wait for feedback from others on this.